includes/Blocks/CorePostTerms.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Core Post Terms Block
+ *
+ * @package WPGraphQL\ContentBlocks\Blocks
+ */
+
+namespace WPGraphQL\ContentBlocks\Blocks;
+
+use WPGraphQL\ContentBlocks\Registry\Registry;
+use WPGraphQL\Model\Term;
+use WP_Block_Type;
+
+/**
+ * Class CorePostTerms.
+ */
+class CorePostTerms extends Block {
+	/**
+	 * String fields for the block.
+	 */
+	public const STRING_FIELDS = [ 'prefix', 'suffix', 'term' ];
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function __construct( WP_Block_Type $block, Registry $block_registry ) {
+		parent::__construct( $block, $block_registry );
+
+		foreach ( self::STRING_FIELDS as $field ) {
+			$this->register_string_field( $field );
+		}
+
+		$this->register_list_of_terms_field();
+	}
+
+	/**
+	 * Registers a string field for the block.
+	 *
+	 * @param string $name The name of the field.
+	 *
+	 * @return void
+	 * @throws \Exception
+	 */
+	protected function register_string_field( $name ) {
+		register_graphql_field(
+			$this->type_name,
+			$name,
+			[
+				'type'        => 'String',
+				'description' => sprintf(
+					// translators: %1$s is the field name, %2$s is the block type name.
+					__( '%1$s of the "%2$s" Block Type', 'wp-graphql-content-blocks' ),
+					ucfirst( $name ),
+					$this->type_name
+				),
+				'resolve'     => static function ( $block ) use ( $name ) {
+					return isset( $block['attrs'][ $name ] ) ? (string) $block['attrs'][ $name ] : null;
+				},
+			]
+		);
+	}
+
+	/**
+	 * Registers a list of terms field for the block.
+	 *
+	 * @return void
+	 * @throws \Exception
+	 */
+	protected function register_list_of_terms_field() {
+		register_graphql_field(
+			$this->type_name,
+			'terms',
+			[
+				'type'        => [ 'list_of' => 'TermNode' ],
+				'description' => __( 'The terms associated with the post.', 'wp-graphql-content-blocks' ),
+				'resolve'     => static function ( $block ) {
```

Comment:
`terms` should be a connection, but if there was justification for it not to be, you'd want to use `$context->get_loader( 'term' )->load_deferred( $term_id )` (on my phone, confirm syntax) instead of manually converting it to a Model.

Reply:
@justlevine Thank you for pointing that out!

You are absolutely right:
`array_map( static fn($term) => $context->get_loader('term')->load_deferred($term->term_id), $terms );`
aligns better with WPGraphQL best practices by leveraging deferred loading for efficiency.

Also, great call on using a connection here! I’ve updated the code accordingly 👍🏻